test/azure/terraform_azure_loadbalancer_example_test.go

```diff
@@ -0,0 +1,96 @@
+// +build azure
+
+// NOTE: We use build tags to differentiate azure testing because we currently do not have azure access setup for
+// CircleCI.
+
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/azure"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTerraformAzureLoadBalancerExample(t *testing.T) {
+	t.Parallel()
+
+	// subscriptionID is overridden by the environment variable "ARM_SUBSCRIPTION_ID"
+	subscriptionID := ""
+	rgName := fmt.Sprintf("terratest-loadbalancer-rg-%s", random.UniqueId())
+	vnetName := fmt.Sprintf("vnet-%s", random.UniqueId())
+	subnetName := fmt.Sprintf("subnet-%s", random.UniqueId())
+	loadBalancer01Name := fmt.Sprintf("lb-public-%s", random.UniqueId())
+	loadBalancer02Name := fmt.Sprintf("lb-private-%s", random.UniqueId())
+	frontendForLB01 := fmt.Sprintf("cfg-%s", random.UniqueId())
+	frontendForLB02 := fmt.Sprintf("cfg-%s", random.UniqueId())
+	publicIPAddressForLB01 := fmt.Sprintf("pip-%s", random.UniqueId())
+	privateIPForLB02 := "10.200.2.10"
```

Comment:
Do we need to specify an IP?  Can't load balancer get one assigned and we output that value in tf code?

Reply:
It is a example to assert against in the tests. The public IP is an example where Azure assigns it.